jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -52,15 +52,20 @@ class Orientation(PluginTemplateMixin, ViewerSelectMixin):
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
     * ``align_by`` (`~jdaviz.core.template_mixin.SelectPluginComponent`)
     * ``wcs_fast_approximation``
-    * ``delete_subsets``
+    * :meth:`delete_subsets`
     * ``viewer``
     * ``orientation``
     * ``rotation_angle``
     * ``east_left``
-    * ``add_orientation``
+    * :meth:`add_orientation`
+    * :meth:`create_north_up_east_left`
+    * :meth:`create_north_up_east_right`
     """
     template_file = __file__, "orientation.vue"
 
+    # defined as traitlet to allow access from UI - leave fixed

```

Comment:
```suggestion
    # defined as traitlet to allow access from UI - leave fixed
    # since this name is used to identify WCS-only layers elsewhere in jdaviz.
```

Reply:
I replaced the comment with: `defined as traitlet in addition to global variable above to allow access from UI - leave fixed` to give more context.  Maybe we should define this in one place and import into others if it is an assumed fixed string, but we can do that later 🤷‍♂️